.github/workflows/pytest.yml

```diff
@@ -23,13 +23,10 @@ jobs:
           else
             echo "No requirements.txt file found, skipping dependency installation."
           fi
-      - name: Test with pytest
-        run: |
-          if [ -f pytest.ini ] || [ -f pyproject.toml ] || [ -d tests ]; then
-            coverage run -m pytest -v -s || echo "Tests failed or skipped."
-          else
-            echo "No test configuration or tests folder found, skipping pytest."
-          fi
+      - name: Downloading pytest
+        run: pip3 install pytest
+      - name: Testing with pytest
+        run: pytest
```

Comment:
Thank you very much for fixing this, this is fantastic!

Reply:
:)